lib/builder/builder.js

```diff
@@ -207,6 +207,8 @@ module.exports = {
 	 * @param {string} parameters.destPath Target path
 	 * @param {boolean} [parameters.cleanDest=false] Decides whether project should clean the target path before build
 	 * @param {boolean} [parameters.buildDependencies=false] Decides whether project dependencies are built as well
+	 * @param {Array} [parameters.includedDependencies=[]] List of build dependencies to be included if buildDependencies is true
```

Comment:
Array of which type? According to the code it's RegExp and strings, right?

Reply:
yes, either RegExp, which will be matched against the project.metadata.name or otherwise compared with a strict comparison (===), so string yes!

Reply:
any change required from my side? all the @param are only annotated with {Array}, or would you like to have it denoted elsewise? Thanks.

Reply:
I think we should document this as `{string[]|RegExp[]}`

Reply:
mhm, this doesn't really fit it... as it implies you can either have a string OR have a RegExp array... I adapted the guidance from https://jsdoc.app/tags-type.html, also see https://github.com/jsdoc/jsdoc/issues/714, and went for {Array.<string|RegExp>}